### PR TITLE
Image break

### PR DESCRIPTION
**Pull request by mwernimont**
Changes made:
-----------
Description

A component with overlay and height dynamic property.  Overlay is set to true automatically, so not required in the code unless unwanted, which then you simply set to false on the component you do not wish an overlay.  Height is default set to 100vh.  Currently using vh as the metric of height, it seems to work well, but all functionality can be tweaked.

Testing:
----------------------------
Before making this pull request, I:
- [ ] Cleaned the code the way Vue likes it - run 'npm run lint --fix'        
- [x] Made sure all tests run
- [ ] Ran WAVE plugin 508 compliance tool

I can confirm this has been checked on:
- [x] Chrome
- [x] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -16,6 +16,9 @@
 <script>
     import WindowSize from "./components/WindowSize";
     import HeaderUSGS from './components/HeaderUSGS';
+    import gsap from "gsap";
+    import { ScrollToPlugin } from "gsap/ScrollToPlugin"; // to trigger scroll events
+    import { ScrollTrigger } from "gsap/ScrollTrigger"; // animated scroll events
     export default {
```
Comment (collnell) [inline src/App.vue:22]: Just for clarification on my end, doing this here imports to the whole app, vs within the component is just for the given section?

**Review by collnell — APPROVED** [review]

**Comment by collnell** [comment]
The structure and purpose of `Visualization.vue` makes so much more sense now.... this is really awesome

File: src/App.vue (hunk 1 above)
Comment (mwernimont) [reply src/App.vue:22]: Well this was a mistake to commit.  I was experimenting, but this brings up a good question.  Can we make the GSAP lib global so we don't have to import in each place we want it?  Will see if we can.

**Comment by mwernimont** [comment]
@collnell  alright found a way to create a global vue instance of the gsap package.  So you can either set a const to this.$root.$gsap and use all the gsap goodness in any component.  The plugin only gets loaded in once but all the components have access to this instance.  I still like the idea of only loading in the individual gsap plugins to the component that needs it, but could be worth future exploration.

**Comment by mwernimont** [comment]
The prototype I deleted here was an experiment, what we are plugging into here is the Vue.use("gsap), thats the magic.

**Comment by mwernimont** [comment]
Sigh I had it backwards, its the prototype. lol. Thats the magic

**Pull request merged**